File: front/src/app/Produto.ts
export class Produto{

    id : string;  
    preco : number
    lugarPartida : string;
    lugarChegada : string;
    urlImg : string;
    dataPartida : string;

    constructor(){
        this.clean();
    }

    clean(): void{
        this.preco = 0;
        this.lugarChegada = "";
        this.lugarPartida = ""
        this.urlImg = "";
        
    }

}

File: api/Models/Produto.cs

using System;
using Microsoft.EntityFrameworkCore;
namespace api.Models
{
    public class Produto
    {

        public long id { get; set; }
    
        public double preco { get; set; }

        public string lugarPartida { get; set; }

        public string lugarChegada { get; set; }

        public string urlImg {get;set;}

        public DateTime dataPartida {get;set;}
        

    }
}